### PR TITLE
UI improvement for the "upcoming session" dialog box.

### LINKED ISSUE #2075: Improve the "No upcoming session dialog" UI
**Actual Behaviour**

The UI for the "No upcoming session dialog" needs improvement.

**Expected Behaviour**

The dialog UI should be improved.

**Steps to reproduce it**

Open LocationsActivity -> Click on any location -> click on upcoming locations icon on the toolbar -> If there is no upcoming session, then the dialog will pop up 

**Screenshots of the issue**

screenshot_2017-12-31-11-55-43-38

**Would you like to work on the issue?**

Yes

**Proposed changes**

Using an alert dialog with a single positive button would be more consistent with the Android design principles.

@iamareebjamal @the-dagger Views ?

### PR DESCRIPTION
**Pull request by SparshWadhwa**
Fixes #2075 

Changes: The ui for the dialog box that comes when user checks upcoming sessions in his location  is improved.
Screenshots for the change: 
whatsapp image 2018-01-07 at 1 40 01 am

### CODE REVIEW

File: android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java [hunk 1]
```diff
@@ -78,6 +78,7 @@
     private RealmDataRepository realmRepo = RealmDataRepository.getDefaultInstance();
     private TextView upcomingSessionText;
     private TextView upcomingSessionTitle;
+    private View upcoming_session_details;
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java]: Use Java notation

File: android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java [hunk 2]
```diff
@@ -174,10 +176,10 @@ public void setUpcomingSession() {
             upcomingSessionText.setText(upcomingTitle);
         } else {
             upcomingSessionTitle.setText(getResources().getString(R.string.no_upcoming_Sess));
-            upcomingSessionText.setVisibility(View.GONE);
+            upcoming_session_details.setVisibility(View.GONE);
         }
     }
-
+//===========================================================================================================
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java]: What's this needed for?

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

File: android/app/src/main/res/layout/upcoming_dialogbox.xml [hunk 3]
```diff
@@ -9,42 +10,43 @@
...
+        android:orientation="horizontal"
+        android:layout_marginTop="@dimen/layout_margin_medium">
 
         <ImageView
             android:id="@+id/track_image_drawable"
             android:layout_width="@dimen/item_image_view_small"
             android:layout_height="@dimen/item_image_view_small"
             android:layout_marginLeft="@dimen/layout_margin_medium"
-            android:layout_marginTop="@dimen/layout_margin_medium"
+
```
Comment (dr0pdb) [inline android/app/src/main/res/layout/upcoming_dialogbox.xml]: Please don't add unnecessary lines.

**Review by dr0pdb — CHANGES_REQUESTED** [review]

File: android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java (hunk 1 above)
Comment (SparshWadhwa) [reply android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java]: ohhk sir.

File: android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java (hunk 2 above)
Comment (SparshWadhwa) [reply android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java]: Actually sir I have added this to distinguish between the different portions of the code. Its for my refrnce. I should remove it before creating pull request  , my bad.

File: android/app/src/main/res/layout/upcoming_dialogbox.xml (hunk 3 above)
Comment (SparshWadhwa) [reply android/app/src/main/res/layout/upcoming_dialogbox.xml]: ohhk sure ,Thanks

File: android/app/src/main/res/layout/upcoming_dialogbox.xml [hunk 4]
```diff
@@ -9,42 +10,42 @@
...
         <TextView
             android:id="@+id/upcoming_session_textview"
             android:layout_width="match_parent"
-            android:textSize="@dimen/text_size_extra_large"
-            android:layout_toRightOf="@id/track_image_drawable"
+            android:layout_height="wrap_content"
             android:layout_marginLeft="@dimen/upcoming_margin_left"
             android:layout_marginRight="@dimen/layout_margin_large"
-            android:layout_marginTop="@dimen/layout_margin_medium"
+            android:layout_toRightOf="@id/track_image_drawable"
```
Comment (iamareebjamal) [inline android/app/src/main/res/layout/upcoming_dialogbox.xml]: You changed RelativeLayout to LinearLayout. `layout_toRightOf` has no meaning in LinearLayout

**Review by iamareebjamal — DISMISSED** [review]

**Comment by iamareebjamal** [comment]
@SparshWadhwa can you please squash the commits so we can merge

**Comment by SparshWadhwa** [comment]
Done sir. @iamareebjamal

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**